android/app/src/main/java/org/fossasia/openevent/adapters/SpeakersListAdapter.java

```diff
@@ -118,14 +120,18 @@ public void onBindViewHolder(RecyclerViewHolder holder, final int position) {
         }
 
 
-        String thumbnail = current.getThumbnail();
+        String thumbnail = Utils.getSpeakerImageUri(current.getThumbnail());
+        if (thumbnail == null)
+            thumbnail = Utils.getSpeakerImageUri(current.getPhoto());
```

Comment:
If thumbnail is null, show the photo field. thumbnail is just smaller version of photo